apps/raptor/lib/raptor_web/views/api_key_view.ex

```diff
@@ -6,4 +6,10 @@ defmodule RaptorWeb.ApiKeyView do
       apiKey: apiKey
     }
   end
+
+  def render("getUserIdFromApiKey.json", %{user_id: user_id}) do
+    %{
+      user_id: user_id
+    }
+  end
```

Comment:
Non-blocking: Would you mind providing context on what utilizes this `ApiKeyView` module?

It's clearly a pattern based on the module contents outside this addition, but after looking through all of `smartcitiesdata`, I don't see any references to `getUserIdFromApiKey` or `ApiKeyView`.

What does this module enable?

Reply:
lib/raptor_web/controllers/api_key_controller.ex contains the method getUserIdFromApiKey. When the Auth0 returns a single user connected to an api_key, we return :
```
render(conn, %{
              user_id: user_id
            })
```

That render return requires a pattern match which is specified here in the api_key_view.ex file